griptape/drivers/sql/amazon_redshift_sql_driver.py

```diff
@@ -0,0 +1,106 @@
+import time
+from typing import Optional
+import boto3
+from griptape.drivers import BaseSqlDriver
+from attr import define, field
+
+
+@define
+class AmazonRedshiftSqlDriver(BaseSqlDriver):
+    database: str = field(kw_only=True)
+    cluster_identifier: str = field(default=None, kw_only=True)
+    workgroup_name: str = field(default=None, kw_only=True)
+    db_user: str = field(default=None, kw_only=True)
+    secret_arn: str = field(default=None, kw_only=True)
+    region_name: str = field(default="us-east-1", kw_only=True)
+    wait_sec: float = field(default=0.3, kw_only=True)
+    client = field(default=None, kw_only=True)
+
+    def __attrs_post_init__(self):
+        if self.client == None:
+            session = boto3.Session(region_name=self.region_name)
+            self.client = session.client("redshift-data")
```

Comment:
If use session, then this becomes unnecessary also.

Reply:
Yes, I'll get rid of the `session` creation here. I plan to keep the `client` as an optional parameter, and create it here if not supplied.

Reply:
No need to supply it here either. Just use a default `Factory` in the field constructor.